llvm/include/llvm/Analysis/CFGPrinter.h

```diff
@@ -148,7 +148,7 @@ std::string CompleteNodeLabelString(
   std::string Str;
   raw_string_ostream OS(Str);
   HandleBasicBlock(OS, *Node);
```

Comment:
Could we rename `Str` to `OutStr` and drop the copy construction?

```suggestion
  std::string OutStr;
  raw_string_ostream OS(OutStr);
  HandleBasicBlock(OS, *Node);
```

Reply:
Applied your feedback to follow-up commit. Thank you!